test/OpenSpaceToolkit/Astrodynamics/GuidanceLaw/ConstantThrust.test.cpp

```diff
@@ -176,138 +154,132 @@ TEST_F(OpenSpaceToolkit_Astrodynamics_GuidanceLaw_ConstantThrust, Getters)
+                ));
+                referencePositionArrayGCRF.add(
+                    Vector3d(referenceRow[1].accessReal(), referenceRow[2].accessReal(), referenceRow[3].accessReal())
+                );
+                referenceVelocityArrayGCRF.add(
+                    Vector3d(referenceRow[4].accessReal(), referenceRow[5].accessReal(), referenceRow[6].accessReal())
+                );
+                referenceManeuverAccelerationArrayGCRF.add(Vector3d(
+                    referenceRow[16].accessReal(), referenceRow[17].accessReal(), referenceRow[18].accessReal()
+                ));
+                referenceMassArray.add(referenceRow[22].accessReal());
+            }
+
+            // Validation loop
+            for (size_t i = 1; i < instantArray.getSize() - 1; i++)
+            {
+                // Current state and instant setup from Orekit test case
+                VectorXd OrekitStateCoordinates(7);
+                OrekitStateCoordinates << referencePositionArrayGCRF[i], referenceVelocityArrayGCRF[i],
+                    referenceMassArray[i];  // Check size input Dynamics
+
+                const VectorXd maneuverContributionFromOrekitStateGCRF =
+                    thrusterDynamicsSPtr->computeContribution(instantArray[i], OrekitStateCoordinates, gcrfSPtr_);
+
+                const double referenceStepSize = 30.0;
+
+                const double maneuverComputationErrorAccelerationContributionXGCRF =
+                    std::abs(maneuverContributionFromOrekitStateGCRF[0] - referenceManeuverAccelerationArrayGCRF[i][0]);
+                const double maneuverComputationErrorAccelerationContributionYGCRF =
+                    std::abs(maneuverContributionFromOrekitStateGCRF[1] - referenceManeuverAccelerationArrayGCRF[i][1]);
+                const double maneuverComputationErrorAccelerationContributionZGCRF =
+                    std::abs(maneuverContributionFromOrekitStateGCRF[2] - referenceManeuverAccelerationArrayGCRF[i][2]);
+                const double maneuverComputationErrorMassContributionGCRF = std::abs(
+                    maneuverContributionFromOrekitStateGCRF[3] -
+                    ((referenceMassArray[i + 1] - referenceMassArray[i]) / referenceStepSize)
+                );
+                const double maneuverComputationAccelerationContributionErrorGCRF =
+                    (maneuverContributionFromOrekitStateGCRF.segment(0, 3) - referenceManeuverAccelerationArrayGCRF[i])
+                        .norm();
+
+                ASSERT_GT(5e-10, maneuverComputationErrorAccelerationContributionXGCRF);
+                ASSERT_GT(5e-10, maneuverComputationErrorAccelerationContributionYGCRF);
+                ASSERT_GT(5e-10, maneuverComputationErrorAccelerationContributionZGCRF);
+                ASSERT_GT(1e-9, maneuverComputationErrorMassContributionGCRF);
+                ASSERT_GT(1e-9, maneuverComputationAccelerationContributionErrorGCRF);
+
+                // Results console output
```

Comment:
Sometimes we just leave them so that its easy to debug if they fail if we change something with the propagator upstream.

I don't mind leaving it since it will spare someone re-adding them later